common/base.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+import logging
+import re
+import os
+
+# Configure logging to write logs to a file
+log = logging.getLogger()
+logging.basicConfig(level=logging.DEBUG, format='%(levelname)s      %(funcName)s.%(filename)s:%(lineno)d %(message)s')
+
+class CliParams:
+    def __init__(self):
+        self.args = None
+        pass
+
+    def get_input_params(self):
+        parser = argparse.ArgumentParser(description="Get Cli Params")
+        parser.add_argument("--test_config", action="store", required=True, help="test_config_path")
+        parser.add_argument("--topology_config", action="store", required=True, help="topology_config_path")
+
+        self.args = parser.parse_args()
+        return vars(self.args) 
+
+def get_acc_mac_vsi(ces_base,acc_iface):
+    acc_ssh=ces_base.get_def_acc_ssh()
+    lines = acc_ssh.exec_command("ifconfig {} | grep ether".format(acc_iface))
+    log.info(lines)
+    print("get_acc_mac_vsi=",lines)
+    acc_mac = lines[0].split()[1]
+    print("acc_mac:",acc_mac)
+
+    #get acc vsi
```

Comment:
Might be a good idea to follow a Python style guide (something like a `Pylint` or `Flake8` linters are popular and will make sure all the script follow the PEP8 format styling.

If you dont want to address this right away, we can look into it as a follow-on PR

Reply:
We will look into a follow on PR